src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -13,7 +13,7 @@
     :class="{ expanded, active: isActive, 'is-group': isGroupMarker }"
     :style="{ '--nesting-index': item.depth }"
     :data-nesting-index="item.depth"
-    :id="`container-${item.uid}`"
+    :id="item.uid ? `container-${item.uid}` : null"
```

Comment:
I think we should keep the `uid` as a requirement since this id value is important for the underlying dynamic scrolling library that is used for nav items.